client/asset/dcr/dcr.go

```diff
@@ -1473,7 +1473,7 @@ func (dcr *ExchangeWallet) AuditContract(coinID, contract dex.Bytes) (asset.Audi
 			contractHash, addr.ScriptAddress())
 	}
 	return &auditInfo{
-		output:     newOutput(txHash, vout, toAtoms(txOut.Value), wire.TxTreeRegular),
+		output:     newOutput(txHash, vout, toAtoms(txOut.Value), txTree),
```

Comment:
If we're sure the tree is always going to be `wire.TxTreeRegular` or we want to enforce that for contracts we audit, then instead of checking both trees in the gettxout call above, we can modify that to only check the regular tree and undo this change.

This is what is done for `Refund`. We only check the regular tree and when creating the prevout, we hardcode wire.TxTreeRegular. But that's likely only safe because we're refunding contracts we created which are currently all regular tree outputs.

Reply:
Are there any circumstances under which this pubkey script would be accepted in a valid stake tree transaction? It's not tagged with a stake opcode, and I don't think it will validate as a valid output in a ticket, vote, or revocation.

Reply:
It's possible, yes. Discussed this with @davecgh on [dex chat](https://matrix.to/#/!SFRQQFIHUUNXARfvew:decred.org/$8rg2i5ekI44QGr3Xn-9SbD50qbAJGusAVbY2AS0tP_0?via=decred.org&via=matrix.org&via=planetdecred.org). Spendable outputs in the stake tree "can be a pay-to-script-hash which could theoretically be anything".

It is possible albeit extremely unlikely for a contract output to be a vote or revocation output i.e. if the user purchased a ticket with a commitment to the contract p2sh address. Another possibility is if a contractor submitted their invoice with an address which itself was actually just funding a swap, i.e. after the treasury vote activates - the payment to that address would be a stake tree tx.